dev/z.ts

```diff
@@ -0,0 +1,26 @@
+// https://github.com/rupa/z
+const completionSpec: Fig.Spec = {
+  name: "z",
```

Comment:
Can we add a description
```suggestion
  name: "z",
  description: "CLI tool to jump around"
```

Reply:
Completely missed it :)